.azure-pipelines/template/sonic-slave-build.yml

```diff
@@ -0,0 +1,69 @@
+parameters:
+- name: arch
+  values:
+  - amd64
+  - armhf
+  - arm64
+- name: distro
+  values:
+  - buster
+  - jessie
+  - stretch
+
+jobs:
+- job: Build_${{ parameters.distro }}_${{ parameters.arch }}
+  pool: sonicbld
+  timeoutInMinutes: 360
+  steps:
+  - template: cleanup.yml
+  - checkout: self
+    clean: true
+  - bash: |
+      set -ex
+
+      USER=`id -un | tr A-Z a-z`
+      SLAVE_DIR=sonic-slave-${{ parameters.distro }}
+      if [ x${{ parameters.arch }} == x"amd64" ]; then
+        SLAVE_BASE_IMAGE=${SLAVE_DIR}
+      else
+        SLAVE_BASE_IMAGE=${SLAVE_DIR}-${{ parameters.arch }}
+      fi
+
+      tmpfile=$(mktemp)
+
+      echo ${{ parameters.arch }} > .arch
+
+      DOCKER_DATA_ROOT_FOR_MULTIARCH=/data/march/docker BLDENV=${{ parameters.distro }} make -f Makefile.work sonic-slave-build | tee $tmpfile
+      SLAVE_BASE_TAG=$(grep "^Checking sonic-slave-base image:" $tmpfile | awk -F ':' '{print $3}')
+      SLAVE_TAG=$(grep "^Checking sonic-slave image:" $tmpfile | awk -F ':' '{print $3}')
+
+      mkdir -p target
+
+      docker tag $SLAVE_BASE_IMAGE-$USER:$SLAVE_TAG local/$SLAVE_BASE_IMAGE-$USER:latest
```

Comment:
Looks like line 42 is no use

Reply:
This will be used for Docker cloud in Jenkins.
Currentlly we can remove it.